test/docker-compose.tpl

```diff
@@ -24,7 +24,7 @@ services:
       args:
         - VCS_REF
         - GRAYLOG_VERSION
-    entrypoint: /usr/bin/tini -- wait-for-it elasticsearch:9200 --  /docker-entrypoint.sh
+    entrypoint: /usr/bin/tini -- wait-for-it elasticsearch:9200 --  /docker-entrypoint.sh server
```

Comment:
So does this mean that all Docker Compose files that use Graylog will have to change? 

When build a new image and use it in our [docker-compose](https://github.com/Graylog2/docker-compose) template, it throws:

```
graylog_1        | wait-for-it: elasticsearch:9200 is available after 6 seconds
graylog_1        | Unknown command -f
```

We can change our file, but what about other users who've made their own docker-compose files? Can it be changed to have a default argument if none is provided?

Reply:
I'm assuming it throws this error because of this line in `docker-entrypoint.sh`

```
    "${GRAYLOG_HOME}/graylog.jar" \
    "$@" \
    -f "${GRAYLOG_HOME}/data/config/graylog.conf"
```

So it was hardcoded before, but now it's an argument. The past behavior doesn't work anymore.

Reply:
I can add something makes it work without the argument to keeps it backwards compatible.

Reply:
should be fixed :+1: thanks for the hint